src/cmd/config.go

```diff
@@ -42,7 +43,8 @@ func GetConfigFromEnvironment() *Config {
 				",",
 			),
 		},
-		ProxyProtocol: envBool("PROXY_PROTOCOL", false),
+		ProxyProtocol:  envBool("PROXY_PROTOCOL", false),
+		CheckerTimeout: envDuration("CHECK_TIMEOUT", 500*time.Millisecond),
```

Comment:
Minor thing, any reason to use different naming here (**Checker**Timeout vs **CHECK**_TIMEOUT)?

Other than that, LGTM.

Reply:
While writing it I was originally looking where to put it in the `HTTPChecker` type (implements `Checker`) so it made sense to call it the `CheckerTimeout`.

And `CHECKER_TIMEOUT` as an EnvVar seemed weird.

I've renamed the setting internally to `CheckTimeout`.